.github/workflows/markdownlint.yml

```diff
@@ -9,12 +9,6 @@ on:
       - ".markdownlint-cli2.jsonc"
       - ".github/workflows/markdownlint.yml"
       - ".github/workflows/markdownlint-problem-matcher.json"
-  pull_request:
```

Comment:
I think we still need this line. We want it to run on PRs, right?

@IEvangelist might know more than I do.